core_java/twitter/src/main/java/ca/jrvs/apps/twitter/DAO/Service.java

```diff
@@ -1,4 +1,4 @@
-package ca.jrvs.apps.DAO;
+package ca.jrvs.apps.twitter.DAO;
```

Comment:
some changes already merged to release. please make sure you can follow gitflow next time.